build_runner/lib/src/package_graph/build_config_overrides.dart

```diff
@@ -34,8 +38,12 @@ Future<Map<String, BuildConfig>> findBuildConfigOverrides(
       throw CannotBuildException();
     }
     final yaml = file.readAsStringSync();
-    final config = BuildConfig.parse(packageGraph.root.name,
-        packageGraph.root.dependencies.map((n) => n.name), yaml);
+    final config = BuildConfig.parse(
+      packageGraph.root.name,
+      packageGraph.root.dependencies.map((n) => n.name),
+      yaml,
+      configYamlSourceUrl: file.path,
```

Comment:
We would need to bump the min version in the pubspec for this.

Reply:
yup!